main.go

```diff
@@ -267,4 +267,12 @@ func setupReconcilers(ctx context.Context, base utils.ReconcilerBase, opts manag
 		setupLog.Error(err, "unable to create controller", "controller", "CloudStackFailureDomain")
 		os.Exit(1)
 	}
+	if err := (&controllers.CksClusterReconciler{ReconcilerBase: base}).SetupWithManager(mgr); err != nil {
+		setupLog.Error(err, "unable to create controller", "controller", "CKSClusterController")
+		os.Exit(1)
+	}
+	if err := (&controllers.CksMachineReconciler{ReconcilerBase: base}).SetupWithManager(mgr); err != nil {
+		setupLog.Error(err, "unable to create controller", "controller", "CKSMachineController")
+		os.Exit(1)
+	}
```

Comment:
could we make this optional? either a flag or some other type of config that enables these controllers
And IMO it should be opt-in for now, so users who are not interested on this don't run these controllers by default